core/vmpu/src/vmpu.c

```diff
@@ -616,10 +616,12 @@ int vmpu_init_pre(void)
 
 void vmpu_init_post(void)
 {
+#if !defined(ARCH_MPU_ARMv8M)
```

Comment:
According to the Cortex M33 user guide, there is no NONBASETHRDENA bit in SCB->CCR (this bit is reserved):
![image](https://user-images.githubusercontent.com/14541274/29667330-9cd63cd6-88e4-11e7-986c-0f199f2fa66a.png)

Reply:
Woah. This is a big concern. If it turns out we can't do `NONBASETHRDENA` on M33, uVisor has to be reengineered a bit. We might come off lucky if `NONBASETHRDENA` is always effectively on by default, though.

Reply:
Looks like the NONBASETHRDENA bit is always 1, so we are good to go.

Reply:
But bit 0 is not NONBASETHRDENA in cortex m33, so how can you have this functionality in v8m?